nbdime/prettyprint.py

```diff
@@ -305,7 +305,7 @@ def pretty_print_metadata(md, known_keys, prefix="", out=sys.stdout):
 
 def pretty_print_output(i, output, prefix="", out=sys.stdout):
     oprefix = prefix+IND
-    t = output.output_type
+    t = output['output_type']
```

Comment:
Why? I've been adding NotebookNode elsewhere to make sure we can use . notation. If you don't want that a lot of prettyprint.py needs to be updated.